<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabChooser</class>
 <widget class="QDialog" name="LabChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose laboratory work</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Laboratory works</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbLw1">
        <property name="text">
         <string>LW1: Basic substitution cyphers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbLw2">
        <property name="text">
         <string>LW2: Frequency Analysis</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbLw3">
        <property name="text">
         <string>LW3: Security policies</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnLaunch">
     <property name="text">
      <string>Launch</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
